packages/eventual-send/src/no-shim.js

```diff
@@ -1,6 +1,10 @@
-// @ts-nocheck
+/* global globalThis */
 import makeE from './E.js';
 
-const hp = HandledPromise;
-export const E = makeE(HandledPromise);
+/** @type {import('./exports.js').HandledPromiseConstructor} */
```

Comment:
I have mixed feelings about the explicit type. We likely could get an implicit type working with a `import from './exports.js'` at the top.

One one hand, it's safer to have an explicit type, on the other we rely on an implicit type for `E`

Reply:
I had incorrect `declare namespace global` instead of `declare global` syntax in `./exports.d.ts`.  With that fixed, the explicit typing was not required, so I removed it.